.copr/yggdrasil.spec.in

```diff
@@ -0,0 +1,80 @@
+%bcond_without check
+
+# https://github.com/redhatinsights/yggdrasil
+%global goipath         github.com/redhatinsights/yggdrasil
+%global commit          @COMMIT@
+
+%gometa -f -i
```

Comment:
Rather than duplicating documentation unrelated to yggdrasil in this spec file, we should propose changes to the Fedora Packaging Guidelines, expanding the documentation about `%gometa`. The macro flags are documented in the macro [definition files themselves](https://pagure.io/go-rpm-macros/blob/master/f/rpm/macros.d/macros.go-srpm#_111) so proposing changes to the upstream documentation should be fairly straightforward.

Reply:
OK